crypto/asn1/bio_asn1.c

```diff
@@ -16,6 +16,7 @@
 #include <string.h>
 #include <internal/bio.h>
 #include <openssl/asn1.h>
+#include "e_os.h"
```

Comment:
I mentioned it once, e_os.h is inappropriate place to keep ossl_assert in. Because e_os.h is meant to be very OS-specific (and even includes quite "heavy" headers such as windows.h). On the other hand ossl_assert is as OS-neutral as it can possibly get. [There are couple of other OS-neutral definitions there.]